storage/namespace.go

```diff
@@ -447,6 +487,18 @@ func (n *dbNamespace) Tick(c context.Cancellable) error {
 
 	wg.Wait()
 
+	// Tick namespaceIndex if it exists
```

Comment:
While you're here, do you mind updating the comment on 462 `// Tick through the shards sequentially to avoid parallel data flushes` to be `// Tick through the shards at a capped level of concurrency`?

Reply:
done